observation_portal/requestgroups/serializers.py

```diff
@@ -739,8 +739,8 @@ def validate(self, data):
             if largest_interval.total_seconds() <= 0:
                 raise serializers.ValidationError(
                     _(
-                        'According to the constraints of the request, the target is never visible within the time '
-                        'window. Check that the target is in the nighttime sky. Consider modifying the time '
+                        'According to the constraints of the request, the target is never visible within the future '
```

Comment:
This just feels like awkward wording.
"The target will not be visible within the time window." Implies that we are only looking at the future without sounding like it was written by a bad sci-fi time cop.